tests/console/consoletest_finish.pm

```diff
@@ -41,12 +42,19 @@ sub run {
     # logout root (and later user) so they don't block logout
     # in KDE
     enter_cmd "exit";
-    $console->reset;
-
-    $console = select_console 'user-console';
-    enter_cmd "exit";    # logout
-    $console->reset;
     wait_still_screen(2);
+    select_console 'user-console';
+    enter_cmd "exit";
+    wait_still_screen(2);
+    unless (check_var('SERIAL_CONSOLE', 0) || check_var('VIRTIO_CONSOLE', 0)) {
```

Comment:
I just used it, same as it's used in `select_serial_terminal`, I assume if it works there then it will also here.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/serial_terminal.pm#L372

Reply:
Sure, sure, it would be just cleaner design. Just a minor detail...